concepts/settings/project-roles.mdx

```diff
@@ -12,15 +12,22 @@ Once a project is created, you can invite others to collaborate:
 
 1. Go to the project settings.
 2. Invite members by entering their email addresses.
-3. Assign roles to each member:
-   - **Admin**: Full access to the project.
-   - **User**: Read-only access to the project.
+3. Assign roles to each member.
 
 <Frame caption="Invite a collaborator">
   <img src="/images/members.jpg" alt="Invite a collaborator" />
 </Frame>
 
-Contact us to create more fine-grained roles.
+### Role Based Access Control (RBAC)
+
+Literal AI supports Role-Based Access Control (RBAC) to streamline and secure access to project resources within teams. RBAC ensures that team members have appropriate permissions aligned with their roles, enhancing both security and efficiency in managing project activities.
+
+Project administrators hold the responsibility of assigning and managing roles for all team members. Each role is designed with a distinct set of permissions, ensuring that users can perform their duties effectively without overstepping into areas beyond their scope.
```

Comment:
IMO this is too verbose and basically just explaining what roles are.

```suggestion
Project administrators hold the responsibility of assigning and managing permissions for all team members. Permissions in Literal AI are packaged as the following set of roles :
```

Reply:
I understand, this is the 3rd time we are revisiting this so we want to make sure that:
- because we are not using industry standard roles we have to explain what each role does.
- not revisiting this section for at least a few cycles.

Reply:
totally agree but it feels like explaining what RBAC means is a bit overkill